gemrb/core/GUI/WindowManager.cpp

```diff
@@ -91,12 +92,14 @@ WindowManager::~WindowManager()
 
 void WindowManager::DestroyWindows(WindowList& list)
 {
-	WindowList::iterator it = list.begin();
-	for (; it != list.end(); ++it) {
-		Window* win = *it;
+	while (!list.empty()) {
```

Comment:
Nope, that doesn't help — still crashes in this loop, with or without the closedWindows.clear() above.